spec/message.abnf

```diff
@@ -54,11 +54,12 @@ local = %s".local"
 match = %s".match"
 
 ; Reserve additional .keywords for use by future versions of this specification.
-reserved-statement = reserved-keyword [s reserved-body] 1*([s] expression)
+reserved-statement = reserved-keyword [s reserved-body-trimmed] 1*([s] expression)
```

Comment:
The naming here is unlike our usual naming. It would be better to follow the `start`/`part` convention and eschew `trimmed`. Something like:

```abnf
reserved-body = reserved-body-start [reserved-body-part]
reserved-body-start = *([s] 1*reserved-body-part)
reserved-body-part = reserved-char / reserved-escape / quoted
```

Note that my version makes an empty body possible and also a 1 character body possible

Reply:
> It would be better to follow the start/part convention

I see what you mean. However, the changes you proposed are a no-op. Let me formulate two alternative proposals.